projects/ngrx.io/tools/examples/shared/package.json

```diff
@@ -16,16 +16,17 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@angular/animations": "^12.0.4",
-    "@angular/cdk": "^12.0.4",
-    "@angular/common": "^12.0.4",
-    "@angular/compiler": "^12.0.4",
-    "@angular/core": "^12.0.4",
-    "@angular/forms": "^12.0.4",
-    "@angular/material": "^12.0.4",
-    "@angular/platform-browser": "^12.0.4",
-    "@angular/platform-browser-dynamic": "^12.0.4",
-    "@angular/router": "^12.0.4",
+    "@angular/animations": "13.0.0",
+    "@angular/cdk": "13.0.0",
+    "@angular/common": "13.0.0",
+    "@angular/compiler": "13.0.0",
+    "@angular/core": "13.0.0",
+    "@angular/forms": "13.0.0",
+    "@angular/material": "13.0.0",
+    "@angular/platform-browser": "13.0.0",
+    "@angular/platform-browser-dynamic": "13.0.0",
+    "@angular/platform-server": "13.0.0",
+    "@angular/router": "13.0.0",
     "@ngrx/component-store": "^12.1.0",
```

Comment:
These NgRx deps and devDeps should also be updated to v13

Reply:
ahh...good catch